pkg/fission-cli/cmd/function/update.go

```diff
@@ -154,12 +154,23 @@ func (opts *UpdateSubCommand) complete(input cli.Input) error {
 		function.Spec.IdleTimeout = &fnTimeout
 	}
 
+	executorType, err := getExecutorType(input)
+	if err != nil {
+		return err
+	}
+
```

Comment:
Just cache the result of the condition for readability and use within the code.

```
isPoolManager = string(executorType) != string(fv1.ExecutorTypePoolmgr)) || (string(function.Spec.InvokeStrategy.ExecutionStrategy.ExecutorType) != string(fv1.ExecutorTypePoolmgr) && string(executorType) != string(fv1.ExecutorTypePoolmgr))

if input.IsSet(flagkey.FnConcurrency) {
        function.Spec.Concurrency = input.Int(flagkey.FnConcurrency)
       if isPoolManager {
          ....
       } 
}

```

Also see if we can avoid duplication of code blocks across with small function at package level.

Reply:
Done